test/unit/v2.1.0/converter-v21-to-v1.test.js

```diff
@@ -1421,6 +1476,7 @@ describe('v2.1.0 to v1.0.0', function () {
                         pathVariables: { pv_foo: 'pv_bar' },
                         pathVariableData: [{ description: null, key: 'pv_foo', value: 'pv_bar' }],
                         rawModeData: null,
+                        graphqlModeData: null,
```

Comment:
Why are PPB tests being affected by this change?

Reply:
This change is because of `retainEmptyValues: true`